js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -653,6 +663,10 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
         return this.option('showCheckBoxesMode') !== 'none';
     },
 
+    _showCustomExpanderIcons: function() {
```

Comment:
This function is named as `show`, but we show nothing here :)
```suggestion
    _hasCustomExpanderIcons: function() {
```

Reply:
good catch )